### DISCUSSION TITLE
Emoji to images?

### DISCUSSION THREAD

**Discussion by honzajavorek** [mkdocs/mkdocs#3082] [Q&A] [answered]
Does anyone know whether there's something like https://github.com/sphinx-contrib/emojicodes for MkDocs?

My idea is that I'd use emoji in my Markdown, but then there would be something that converts it to images, so that they display the same way to anyone seeing the HTML in their browser. Basically preventing any accessibility issues emojis might have, or the known issues where one emoji looks different on different systems, thus leading to different meanings.

**Comment by pawamoy** [comment, marked as answer]
Try the pymdownx.emoji extension :)

**Reply by honzajavorek** [reply]
TIL https://facelessuser.github.io/pymdown-extensions/, wow! Thanks, this is helpful!

**Comment by Andre601** [comment]
If you're using the mkdocs-material theme does it ship with the `pymdownx.emoji` package which adds emojis in SVG format for you to use.

**Reply by honzajavorek** [reply]
Thanks, I don't use the theme, I have a custom one, but `pymdownx.emoji` looks like installable separately.
